bahmni-emr/build.gradle

```diff
@@ -1,26 +1,113 @@
 plugins {
     id "nebula.ospackage" version "3.4.0"
+    id "java"
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+    maven {
+        url 'http://mavenrepo.openmrs.org/nexus/content/repositories/public'
+    }
+    maven {
+        url 'https://oss.sonatype.org/content/repositories'
+    }
 }
 
 group 'org.bahmni.emr'
 version project.bahmniRelease
 
 task extractOmods(type: Copy) {
-    from zipTree(file("${projectDir}/resources/distro-0.91-SNAPSHOT-distro.zip"))
```

Comment:
We need to hold on this specific change since we are pointing to bahmni distro 0.91 for release 0.92. @shilpa-iplit @binduak

Reply:
Please check the second commit. I have corrected the mistake. @SwathiVarkala  @binduak